From 10e9cc6fa4e022b06f57384404774e1e65e1b666 Mon Sep 17 00:00:00 2001
From: Phong Tran <540580+phongse@users.noreply.github.com>
Date: Sun, 1 Jun 2025 20:55:54 +0200
Subject: [PATCH 1/2] bump `go` version to `1.24.3`

---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 6e126b7..89544da 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/qdm12/ddns-updater
 
-go 1.23.0
+go 1.24.3
 
 require (
 	github.com/breml/rootcerts v0.2.19
-- 
2.49.0

